src/chrome/content/rules/Rb-media-group.de.xml

```diff
@@ -0,0 +1,59 @@
+<!--
+	Nonfunctional hosts in *.rb-media-group.de:
+	http redirect
+        apc4.rb-media-group.de
+        ats2.rb-media-group.de
+        backups1.rb-media-group.de
+        buero-w.rb-media-group.de
+        esxi.rb-media-group.de
+        gateway.rb-media-group.de
+        hostmaster.rb-media-group.de
+        liveconfig.rb-media-group.de
+        live-support.rb-media-group.de
+        ns1.rb-media-group.de
+        ns5.rb-media-group.de
+        phpmyadmin.rb-media-group.de
+        rbbackups1.rb-media-group.de
+        robot.rb-media-group.de
+        sitemap.rb-media-group.de
+        static-v13.rb-media-group.de
+        statistik.rb-media-group.de
+        support.rb-media-group.de
+        switch-s2.rb-media-group.de
+        transfer.rb-media-group.de
+        v13.rb-media-group.de
+        webmail.rb-media-group.de
+        vertrieb.rb-media-group.de
+	connection refused
+        autodiscover.rb-media-group.de
+        community.rb-media-group.de
```

Comment:
WFM

Reply:
What does that mean sorry?

Reply:
This host works for me over HTTPS, so please consider adding it as target.